### ISSUE TITLE
Font missing in docker container

### ISSUE THREAD

**Issue by grst** [GregorySchwartz/too-many-cells#3]
Sorry for bothering you again... 
In the "Painting tree" step the docker container fails because of a font missing: 

```
> sudo docker run -it -v /storage/scratch/toomanycells:/scratch gregoryschwartz/too-many-cells:0.1.1.0 make-tree -m /scratch/harmony_toomanycells.csv --no-filter --normalization NoneNorm  --dense
Painting sketches [===================================>..............]  70%
Clumpiness requires labels for cells, skipping...
Painting tree [===========================================>..........]  80%
too-many-cells: /opt/build/.stack-work/install/x86_64-linux/lts-12.0/8.4.3/share/x86_64-linux-ghc-8.4.3/SVGFonts-1.6.0.3/fonts/LinLibertine.svg: openFile: does not exist (No such file or directory)       
```

**Comment by GregorySchwartz** [comment]
Wow, nice catch! I usually use labels so I never get to see this issue which only appears in the docker without a labels file. I fixed it, confirmed working on my end. I needed to copy the font svg over and change the locale of the docker image. I updated the docker image version 0.1.1.0, so pull that. Let me know if everything goes well! Fixed in https://github.com/GregorySchwartz/too-many-cells/commit/e1abd8cf846f082d4e2799e615636db09588631e.
